### ISSUE TITLE
Let administrators add the default presentation to BBB rooms even if the teacher uploaded another one

### ISSUE THREAD

**Issue by abias** [blindsidenetworks/moodle-mod_bigbluebuttonbn#252]
Admins are able to provide a default presentation file with the `mod_bigbluebuttonbn | presentationdefault` setting which will be added to all BBB rooms.

However, teachers are also able to provide a presentation file within a BBB activity instance.
As soon as a teacher uploads a presentation file in his BBB activity instance, the default presentation is not added to the BBB room anymore.

I would like to propose to let administrators add the default presentation to BBB rooms even if the teacher uploaded another one.

User story:
As an admin, I have created a nice default presentation which shows some basic instructions and information about our BBB instance on the first slide and provides several additional blank slides which can be used as a whiteboard background.
I want to be sure that this default presentation is uploaded and available within the presentation picker in all BBB rooms even if the teacher decided to upload a presentation of his own.

Note:
This might best be solved after #251 where the support to upload multiple presentation files to a BBB room was requested.

**Comment by jfederico** [comment]
Yes, this feature does not make sense if the support for multiple presentations doe not exist as the override would make the optionf for teachers useless.

One alternative in your particular case would be to disable the functionality for teachers by adding a checkbox to administrators for "force default presentation on every session"
